tag/map_codec_test.go

```diff
@@ -70,7 +72,7 @@ func Test_EncodeDecode_Set(t *testing.T) {
 				{k1, "v1"},
 				{k2, "v2"},
 				{k3, "v3"},
-				{k4, "v4 is very weird <>.,?/'\";:`~!@#$%^&*()_-+={[}]|\\"},
+				{k4, "v4_"},
```

Comment:
The values don't get escaped, so it's probably a good idea to keep the weird string we had here before.

Reply:
Done.